frontend/components/SummaryTable.js

```diff
@@ -0,0 +1,76 @@
+const SummaryTable = (props) => {
+    const yearData = props.data.data['2018'];
+    const percentAssisted = `${(yearData["% of Households below 200% FPL Assisted"] * 100).toFixed(2)}%`;
+    const selectedYear = '2018';
```

Comment:
Can you move this to the top and then use it to get the year data so we only hardcode this once.

` const yearData = props.data.data[selectedYear];`